src/auth/auth.service.ts

```diff
@@ -15,25 +15,36 @@ import {
   WRONG_EMAIL,
   USER_NOT_FOUND,
   AUTHENTICATED_ERROR,
+  LogType,
 } from '../constants';
 import { mailer } from '../nodemailer';
 import * as dotenv from 'dotenv';
 import * as env from 'env-var';
 import { User } from '../user/user.model';
+import { LogService } from '../log/log.service';
 dotenv.config();
 
 @Injectable()
 export class AuthService {
   constructor(
     private userService: UserService,
     private jwtService: JwtService,
+    private logService: LogService
   ) {}
 
   async login(userDto: CreateUserDto) {
     const user = await this.validateUser(userDto);
     if (!user.registered) {
       throw new HttpException(AUTHENTICATED_ERROR, HttpStatus.FORBIDDEN);
     }
+    // log to mongo
+    const log = {
+      message: `user: ${user.email} was login`,
+      where: 'auth.servise.ts (login())',
+      type: LogType[LogType.update]
```

Comment:
why don't you use numbers? If you want strings to be stored as logtype, then change the enum to use strings instead of numbers